recipes/uproot-browser/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "uproot-browser" %}
+{% set version = "0.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/uproot_browser-{{ version }}.tar.gz
+  sha256: c1fdeeec79c74751a1520a333f2e0df83eec0715cc371ac70ddc9910e96f3441
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - uproot-browser = uproot_browser.__main__:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - flit-core
```

```suggestion
    - flit-core >=3.3
```
https://github.com/scikit-hep/uproot-browser/blob/5c28fd2290fcbb02ba39ac18a7d6677402591084/pyproject.toml#L2